### ISSUE TITLE
Installer: allow different installation directories

### ISSUE THREAD

**Issue by catastrop** [denoland/deno#2546]
can i use `/usr/local/bin` instead of `$HOME/.deno/bin`

**Comment by bartlomieju** [comment]
+1 for this idea, I propose to utilize this syntax:
```
deno install --dir /usr/local/bin file_server https://deno.land/std/http/file_server.ts
deno install -d /usr/local/bin file_server https://deno.land/std/http/file_server.ts
```

Although it seems trivial I believe it vets the need for more robust CLI parser in standard lib - ie. Clap counterpart in TS.

**Comment by bartlomieju** [comment]
This feature landed in #2551

CC @ry

**Comment by bartlomieju** [comment]
CC @ry this issue can be closed now
